packages/gateway/src/features/registration/root-resolvers.test.ts

```diff
@@ -1796,6 +1796,35 @@ describe('Registration root resolvers', () => {
       )
     })
 
+    it('throws error if no task entry found given id', () => {
+      const compositionID = 'cd168e0b-0817-4880-a67f-35de777460a5'
+      fetch.mockResponseOnce(
+        JSON.stringify({
+          resourceType: 'Bundle',
+          id: 'd2ca298f-662f-4086-a8c5-697517a2b5a3',
+          meta: {
+            lastUpdated: '2018-12-13T04:02:42.003+00:00'
+          },
+          type: 'searchset',
+          total: 0,
+          link: [
+            {
+              relation: 'self',
+              url: 'http://localhost:3447/fhir/Task?focus=Composition/cd168e0b-0817-4880-a67f-35de777460a5s'
+            }
+          ],
+          entry: []
+        })
+      )
+      expect(
```

Comment:
This test will never fail if you do not return / await this call. It's a good practice to make sure a test can both succeed and fail while writing it

Reply:
Got it. I have added return before expect